src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -241,255 +245,281 @@ export default function NonProfitApply() {
+                      <CalendarMonthIcon />
+                      {START_DATE} to {END_DATE}
+                    </div>
+                    <div className="profile__header">
+                      <PlaceIcon />
+                      {LOCATION}
+                    </div>
+                  </Grid>
+                </Grid>
                 <br />
-
               </div>
             </div>
           </div>
-          {
-            user && loading && (
-              <DetailsContainer container>
-                Loading ...
-                <Puff stroke="#0000FF" />
-                <Puff stroke="#0000FF" />
-              </DetailsContainer>
-            )
-          } 
+          {user && loading && (
+            <DetailsContainer container>
+              Loading ...
+              <Puff stroke="#0000FF" />
+              <Puff stroke="#0000FF" />
+            </DetailsContainer>
+          )}
         </DescriptionStyled>
-        
       </DetailsContainer>
 
-      
-      <DetailsContainer container>   
-                     
-        <DescriptionStyled>
-          
-          <p>
-            <Typography style={style}><b>Name of Charity Organization:</b></Typography>
+      <FormContainer>
+        <DetailsContainer container>
+          <DescriptionStyled>
+            {/* <hr /> */}
+            <Typography variant="h4" sx={{ color: "#1976d2" }}>
+              This form helps us to find the charities that are the right fit
+              for our event.
+            </Typography>
+            <Typography variant="h5">
+              If you don't have the time right now just fill in your quick
+              thoughts and come back later to edit.
+            </Typography>
             <br />
-            <Typography style={style}>We support charities including non-profits (NPOs) and non-government
-            organizations (NGOs).</Typography>
-          </p>
-           
-        
-        {nonProfitOptions.length > 0    && <Autocomplete
-              disablePortal
-              value={formState.charityName}
-              onChange={(event, newValue) => {                
-                ReactPixel.track('NPO: Charity Name', {
-                  content_name: 'Nonprofit Application',
-                  status: newValue,
-                  content_category: 'Nonprofit Application',
-                  value: 0.02,
-                  currency: 'USD',
-                });
+            <p>
+              <b>Name of Charity Organization:</b>
 
-                ga.event({
-                  action: 'NPO: Charity Name',
-                  params: {
-                    content_name: 'Nonprofit Application',
+              <br />
+              <Typography style={style}>
+                We support charities including non-profits (NPOs) and
+                non-government organizations (NGOs).
+              </Typography>
+            </p>
+            {nonProfitOptions.length > 0 && (
```

Comment:
We want to always show Autocomplete even if we don't have something returned here - otherwise someone won't be able to add their nonprofit name

Reply:
I apologize. This is again the prettier reformatting code. I will make sure to uninstall the prettier from the next commit.

Reply:
All good - it works well if it's just you, but doesn't scale very well unless your entire team is using it

Reply:
I will make a note of this. Thanks Greg.